File: SeekMatch-Client/src/app/shared/models/bookmark.ts
import { JobOffer } from "./job-offer";

export class Bookmark {
    public id: string | undefined = undefined;
    public jobOfferId: string = '';
    public jobOffer: JobOffer | null = null;
    public createdAt: string | null = null;
    public talentId: string = '';
}

File: SeekMatch-Server/SeekMatch.Core/Entities/Talent/Bookmark.cs
namespace SeekMatch.Core.Entities
{
    public class Bookmark : BaseEntity
    {
        public JobOffer? JobOffer { get; set; }
        public required string JobOfferId { get; set; }
        public Talent? Talent { get; set; }
        public required string TalentId { get; set; }
    }
}
